io.openems.edge.timeofusetariff.entsoe/src/io/openems/edge/timeofusetariff/entsoe/ExchangeRateApi.java

```diff
@@ -0,0 +1,35 @@
+package io.openems.edge.timeofusetariff.entsoe;
+
+import java.io.IOException;
+
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+
+public class ExchangeRateApi {
+
+	private static final String BASE_URL = "https://api.exchangerate.host/latest?base=%s";
```

Comment:
Should add a note here and in the readme.

This is a text from the API response:
"If you or your company use this project or like what we doing, please consider backing us so we can continue maintaining and evolving this project." -> "https://exchangerate.host/#/donate"